server.js

```diff
@@ -18,9 +33,36 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+// Get the 20 most recent thoughts
+app.get('/', async (req, res) => {
+  const thoughts = await Thought.find().sort({ createdAt: 'desc' }).limit(20).exec()
+  res.json(thoughts)
+})
+
+// Send a new thought
+app.post('/', async (req, res) => {
+  const { message } = req.body
+  const thought = new Thought({
+    message: message,
+  })
+  try {
+    const savedThought = await thought.save()
+    res.status(201).json(savedThought)
+  } catch (err) {
+    res.status(400).json({ message: 'Could not save happy thought to database', error: err.errors })
+  }
+})
+
+// Like a thought
+app.post('/:id/like', async (req, res) => {
```

Comment:
Update this endpoint so it corresponds with the documentation:

### `POST /:thoughtId/like`

This endpoint doesn't require a JSON body. Given a valid thought id in the url, the API should find that thought, and update its `hearts` property to add one heart.